_people/TahiraReidSmith.md

```diff
@@ -0,0 +1,12 @@
+---
+first_name: Tahira
+last_name: Reid Smith
+title: Associate Professor of Mechanical Engineering
+school: Purdue University
+website: https://engineering.purdue.edu/ME/People/ptProfile?resource_id=74127
+image: https://engineering.purdue.edu/ResourceDB/ResourceFiles/image74131
+---
+Prof. Reid Smith leads the [Research in Engineering and Interdisciplinary Design (REID) Lab](https://engineering.purdue.edu/reidlab/) at Purdue University. She studies how to integrate human-centered design into human-machine systems.
+- Akash, Kumar, et al. "[Human Trust-based Feedback Control: Dynamically varying automation transparency to optimize human-machine interactions.](https://arxiv.org/pdf/2006.16353.pdf)" arXiv preprint arXiv:2006.16353 (2020).
+- Hu, Wan-Lin, et al. "[Real-time sensing of trust in human-machine interactions.](https://www.researchgate.net/profile/Kumar_Akash/publication/312545797_Real-Time_Sensing_of_Trust_in_Human-Machine_Interactions/links/59ee290daca272029ddf65a9/Real-Time-Sensing-of-Trust-in-Human-Machine-Interactions.pdf)" IFAC-PapersOnLine 49.32 (2016): 48-53
```

Comment:
Is there a better link than researchgate.net

Reply:
Their lab site linked to a pdf hosted directly by Purdue, which I included. There was also a version on Science Direct if that'd be preferable.